### PR TITLE
Issue #1404 implement tests for ACL layer 

### PR DESCRIPTION
**Pull request by youkofan**

### CODE REVIEW

File: api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.ContextConfiguration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = {AclTestBeans.class, AclSecurityConfiguration.class})
```
Comment (ekazachkova) [inline api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java]: Could we move it to `AclTestConfiguration`?

File: api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+public class LogApiServiceTest extends AbstractAclTest {
+
+    @Autowired
+    private LogApiService logApiService;
+
+    @Autowired
+    private LogManager logManager;
+
+    private LogPagination.LogPaginationBuilder logPagination;
```
Comment (ekazachkova) [inline api/src/test/java/com/epam/pipeline/acl/log/LogApiServiceTest.java]: Could we initialize this field? (as it done for `logFilter`)

File: api/src/main/java/com/epam/pipeline/controller/region/CloudRegionController.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
 import com.epam.pipeline.entity.info.CloudRegionInfo;
 import com.epam.pipeline.entity.region.AbstractCloudRegion;
 import com.epam.pipeline.entity.region.CloudProvider;
-import com.epam.pipeline.manager.region.CloudRegionApiService;
+import com.epam.pipeline.acl.region.CloudRegionApiService;
```
Comment (ekazachkova) [inline api/src/main/java/com/epam/pipeline/controller/region/CloudRegionController.java:25]: Could we move `CloudRegion` changes into separate PR?

File: api/src/test/java/com/epam/pipeline/test/acl/TestAclSecurityConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (ekazachkova) [inline api/src/test/java/com/epam/pipeline/test/acl/TestAclSecurityConfiguration.java]: Why we need this class?

**Review by ekazachkova — CHANGES_REQUESTED** [review]
Also, could you please provide PR description

**Comment by youkofan** [comment]
I'm cloed the PR, because here is the current version in another branch - https://github.com/epam/cloud-pipeline/pull/1442

**Pull request closed without merging**